adoc/extensions/sycl_khr_queue_size_queries.adoc

```diff
@@ -0,0 +1,106 @@
+[[sec:khr-queue-size-queries]]
+= sycl_khr_queue_size_queries
+
+This extension allows developers to query the size queues, meaning the number of
+nonfinished commands submitted to this particular queue.
+
+{note} Since commands are executed asynchronously, the returned value is a
+snapshot in time {endnote}
+
+{note} The behavior of [code]#size()# is associated with the SYCL [code]#queue#
+objets, and and not with the underlying <<native-backend-object>> {endnote}
+
+[[sec:khr-queue-size-queries-dependencies]]
+== Dependencies
+
+This extension has no dependencies on other extensions.
+
+[[sec:khr-queue-size-queries-feature-test]]
+== Feature test macro
+
+An implementation supporting this extension must predefine the macro
+[code]#SYCL_KHR_QUEUE_SIZE_QUERIES# to one of the values defined in the table
+below.
+
+[%header,cols="1,5"]
+|===
+|Value
+|Description
+
+|1
+|Initial version of this extension.
+|===
+
+
+[[sec:khr-queue-size-queries-funct]]
+== Queue Functions added to query size information
+
+
+This extension adds the following functions to the [code]#sycl::queue# clase,
+which provides information about the size of the queue.
+
+'''
+
+.[apidef]#khr::queue::empty#
+[source,role=synopsis,id=api:khr-queue-size-queries-empty]
+----
+bool empty() const
```

Comment:
```suggestion
bool khr_empty() const
```

We prefix new member functions with "khr_".  See the sycl_khr_default_context KHR.

Reply:
Thanks!